lib/isolator/adapters/ethon_adapter.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+if defined?(::Ethon::Easy)
```

Comment:
Let's move all checks to `isolator.rb` to avoid tons of `require` ('cause it slows down gem loading process). I mean in the end of the file:

```ruby
require "isolator/adapters/ethon_adapter" if defined?(::Ethon::Easy)
# etc
```

Also, let's group adapters by type, i.e. `adapters/http/ethon`, etc